gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,69 @@
+//tag::intro1[]
+
+Flex Gateway includes the following CLI command to test gateway health. You can run this command manually on any Flex Gateway:
```

Comment:
```suggestion
Flex Gateway includes the following CLI command to test gateway health:
```
Not mentioning that they can also run the command automatically implies it's not possible to do so, at least at first glance. I think it's better to omit mention to manually running, and mention both together below (see another suggestion.)